homeassistant/components/smartthings/sensor.py

```diff
@@ -526,3 +541,50 @@ def state(self):
             return three_axis[self._index]
         except (TypeError, IndexError):
             return None
+
+
+class SmartThingsPowerConsumptionSensor(SmartThingsEntity, SensorEntity):
+    """Define a SmartThings Sensor."""
+
```

Comment:
You're not setting state class or last reset. We need those to be useful for energy management. https://developers.home-assistant.io/docs/core/entity/sensor#long-term-statistics

Reply:
Ah that's because of the `end` being set, and so it not including current data.